features/dnsresolver/netty/src/test/java/org/opennms/netmgt/dnsresolver/netty/NettyDnsResolverCircuitBreakerIT.java

```diff
@@ -56,6 +56,10 @@ public void canTriggerOpenCircuit() throws InterruptedException, TimeoutExceptio
         NettyDnsResolver dnsResolver = new NettyDnsResolver(eventForwarder, new MetricRegistry());
         // Use a non-routable address as the target - we want the queries to fail due to timeouts
         dnsResolver.setNameservers(InetAddressUtils.str(InetAddressUtils.UNPINGABLE_ADDRESS));
+        dnsResolver.setBreakerFailureRateThreshold(80);
```

Comment:
Move the defaults to the class instead of having every user call the setters.

Reply:
Done